package.json

```diff
@@ -6,28 +6,41 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview",
-    "prepare": "husky install"
+    "preview": "vite preview"
   },
   "dependencies": {
+    "atomic-router": "^0.8.0",
+    "atomic-router-react": "^0.8.0",
+    "classnames": "^2.3.2",
     "commitizen": "^4.2.5",
     "@ant-design/icons": "^4.8.0",
     "antd": "^5.0.2",
     "effector": "^22.4.0",
     "effector-react": "^22.3.4",
+    "effector-forms": "^0.0.24",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "history": "^5.3.0",
+    "lodash": "^4.17.21",
+    "patronum": "^1.12.3"
   },
   "devDependencies": {
     "commitizen": "^4.2.5",
     "@commitlint/cli": "^17.3.0",
+    "@types/lodash": "^4.14.189",
+    "@types/node": "^18.11.9",
     "@types/react": "^18.0.24",
     "@types/react-dom": "^18.0.8",
     "@vitejs/plugin-react": "^2.2.0",
     "cz-customizable": "^7.0.0",
+    "lint-staged": "^13.0.3",
+    "eslint": "^8.28.0",
+    "prettier": "^2.7.1",
+    "sass": "^1.56.1",
```

Comment:
sass is not needed, we will use css in js

Reply:
Thank you. deleted